chpl/chpl-api/e2e/collections/activity-controller.postman_collection.json

```diff
@@ -0,0 +1,333 @@
+{
+	"info": {
+		"name": "activity-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET /activity/metadata/atls/{id}",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"var ReqBody= {\r",
+							"        userName:pm.environment.get(\"emailONCStaff\"),password:pm.environment.get(\"passwordONCStaff\")\r",
+							"    };\r",
+							"pm.sendRequest({\r",
+							"    url: pm.environment.get(\"authurl\"),\r",
+							"    method: \"POST\",\r",
+							"    header: {\r",
+							"        \"Content-Type\": \"application/json\",\r",
+							"        \"API-KEY\": pm.environment.get(\"apiKey\")\r",
+							"    },\r",
+							"    body: JSON.stringify(ReqBody)\r",
+							"}, function (err, response) {\r",
+							"   var jsonData = response.json();\r",
+							"   pm.environment.set(\"token\",jsonData[\"token\"])\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"ONC_STAFF users should get 200 status and response is not null for /activity/metadata/atls/{id} end point\", function () {\r",
+							"    const exprespbody = pm.response.json();\r",
```

Comment:
Can we change "exprespbody" to be more meaningful name? As it is actual response body, name can eb something like actualResponseBody or actualBody

Similar comment to all requests below.